.github/CODEOWNERS

```diff
@@ -517,6 +517,7 @@ pep-0659.rst  @markshannon
 pep-0660.rst  @pfmoore
 pep-0661.rst  @taleinat
 pep-0662.rst  @brettcannon
+pep-0662-editables.json  @brettcannon
```

Comment:
Can we put the file in a `pep-0662` subdirectory? I know the repo is inconsistent, but I would like to try and move towards keeping ancillary files in subdirectories.

Reply:
Done.